pkgs/tools/wayland/hyprevents/default.nix

```diff
@@ -0,0 +1,40 @@
+{ fetchgit, lib, makeWrapper, stdenv, hyprland, bash }:
+
+stdenv.mkDerivation rec {
```

Comment:
Nit: Prefer `finalAttrs` over `rec`

Documentation: https://nixos.org/manual/nixpkgs/unstable/#mkderivation-recursive-attributes
Example: https://github.com/NixOS/nixpkgs/compare/17f96f7b978e61576cfe16136eb418f74fab9952..9e6ea843e473d34d4f379b8b0d8ef0425a06defe

Reply:
Note: This is actually an unused `rec`, so you can remove `rec` without adding `finalAttrs`.